internal/jobservice/repository/postgres.go

```diff
@@ -86,6 +68,10 @@ func (s *JSRepoPostgres) Setup(ctx context.Context) {
 			panic(err)
 		}
 	}
+
+	if s.jobServiceConfig.PurgeJobSetTime > 0 {
+		go s.purgeExpired(ctx)
```

Comment:
Is `Setup` the right place to start this routine? My initial read is that `Startup` would be a better place for this to live.